### DISCUSSION TITLE
Project status

### DISCUSSION THREAD

**Discussion by ernado** [open2b/scriggo#930] [General]
Hey, I see activity decline in repo, everything is alright?

Any help is needed?

Project is really great and undervalued, one of my favorite go projects overall ❤️

**Comment by gazerro** [comment]
We are currently busy working on another internal project using Scriggo. We welcome contributions also including blog posts and documentation.

> Project is really great and undervalued, one of my favorite go projects overall

Thanks, we appreciate it very much.

**Comment by zapateo** [comment]
For anyone reading this discussion in 2026 or later, we've released a [new version of Scriggo, the v0.61.0](https://github.com/open2b/scriggo/discussions/1015). 🎉
